### PR TITLE
docs: translate `docs/roadmap.md` into Chinese

### PR DESCRIPTION
**Pull request by htmlin**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,150 +1,150 @@
...
-sidebar_label: Roadmap
+title: Babel 路线图
+sidebar_label: 路线图
 ---
 
-This document outlines some of the improvements our team members would like to work on during this year.
+本文档概述了我们的团队成员希望在今年进行的一些改进。
 
-This is far from being a complete list of all the new features or important changes that we'll bring to Babel, but it's a good summary if you are interested in the general direction that the project is moving toward. We may not actually finish every listed point or may delay some of them to the next year. Some of them have clear starting and ending points, while others need more research or [RFCs](https://github.com/babel/rfcs).
+这远非我们将为 Babel 带来的所有新功能或重要更改的完整列表，但是，如果你对项目发展的大方向感兴趣，那么这是一个很好的总结。实际上，我们可能不能完成所有列出的要点，或者可能会将其中一些推迟到明年。其中一些有明确的起点和终点，而另一些则需要更多的研究或 [RFCs](https://github.com/babel/rfcs)。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
这并不是我们将为 Babel 带来的所有新功能或重要更改的完整列表，但是，如果你对项目发展的大方向感兴趣，那么这是一个很好的总结。实际上，我们也许并不能完成列出的所有要点，甚至可能会将某些实现推迟到明年。某些要点已有明确的起止时间，而另一些则需要更多的研究或 [RFCs](https://github.com/babel/rfcs)。
```

File: docs/roadmap.md [hunk 2]
```diff
@@ -1,150 +1,150 @@
...
 
-If your company is interested and would like to directly sponsor any particular item please [reach out](mail:team@babeljs.io)!
+如果贵公司有兴趣并希望直接赞助任何项目，请 [联系我们](mail:team@babeljs.io)！
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
如果贵公司有兴趣并希望直接赞助，请 [联系我们](mail:team@babeljs.io)！
```

File: docs/roadmap.md [hunk 3]
```diff
@@ -1,150 +1,150 @@
...
 
-## Babel 2021 Roadmap
+## Babel 2021 路线图
 
 ### Babel 8
 
-We have been talking about the Babel 8 release for more than one year (we initially scheduled it about one year ago)! However, we are now closer then ever to it's release!
+关于 Babel 8 的发布我们已经讨论了一年多了（我们最初是在一年前安排的）！不过，我们现在比以往任何时候都离它的发布更近！
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
关于 Babel 8 的发布我们已经讨论了一年多了（我们早在一年前就有了安排）！不过相比之前，我们现在离它的发布越来越近了！
```

File: docs/roadmap.md [hunk 4]
```diff
@@ -1,150 +1,150 @@
...
-- We would like to release Babel as a pure ESM package. We are now in the process of converting our sources to be compatible with Node.js' ESM implementation, and while doing so, we are examining how we can make it easier for people that currently use Babel to compile ESM to CJS.
-- We are trying to align our TypeScript AST with the [`typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint/) project. Our ASTs are _almost_ identical, but we need to introduce some small breaking changes to fully align.
-- Our release infrastructure doesn't support yet pre-releases, or using multiple "main" branches (one for Babel 8 and one for Babel 7).
-- We haven't figured out yet a policy for Babel 7 maintenance.
+剩下的大部分工作都是在 [跟踪 issue](https://github.com/babel/babel/issues/10746)，但仍有一些阻塞内容：
+- 我们希望取消对 [Node.js 10](https://github.com/nodejs/Release) 的支持，它将在 2021-04-30 停止维护。
+- 我们希望将 Babel 作为一个纯 ES 模块包发布。现在我们正在转换我们的源码以兼容 Node.js 的 ES 模块实现，同时，我们正在研究如何让目前使用 Babel 的人更容易地将 ES 模块编译成 CommonJS 模块。
+- 我们正在尝试将我们的 TypeScript AST 与 [`typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint/) 项目对齐。我们的 AST _几乎_ 完全相同，但我们还需要引入一些小的突破性变化才能完全一致。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
- 我们正在尝试将我们的 TypeScript AST 项目与 [`typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint/) 项目对齐。它们的 AST _几乎_ 完全相同，但我们还需要做出一些小的突破性更改，才能完全一致。
```

File: docs/roadmap.md [hunk 5]
```diff
@@ -1,150 +1,150 @@
...
+- 我们的发布基础设施还不支持预发布，也不支持使用多个 "main" 分支（一个用于 Babel 8，一个用于 Babel 7）。
+- 我们还没有弄清楚 Babel 7 的维护策略。
 
-### Implement new TC39 proposals
+### 实现新的 TC39 提案
 
-Babel can currently parse all the Stage 3 proposals, and we can transform all of them except for top-level await, import assertions and JSON modules (which are best handled by bundlers working with the dependencies graph).
+Babel 目前可以解析所有第 3 阶段提案，除了顶级 await、导入断言和 JSON 模块（它们最好由使用依赖关系图的构建工具来处理）之外，我们都可以转译它们。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
除了顶级 await、导入断言和 JSON 模块（它们最好由使用依赖关系图的构建工具来处理）之外，Babel 目前可以解析并转译所有第 3 阶段提案。
```

File: docs/roadmap.md [hunk 6]
```diff
@@ -1,150 +1,150 @@
...
-- Transform for the Module Blocks proposal (we implemented parsing in Babel 7.13.0).
+我们支持所有第 2 阶段提案，除了：
+- 新版装饰器提案（我们需要同时实现解析和转译）；
+- 转译块级模块提案（在 Babel 7.13.0 我们实现了解析）。
 
-We will implement support for decorators, and investigate if and how we can implement a transform for module blocks.
+我们将实现对装饰器的支持，并研究是否应该以及如何实现对块级模块的转译。
 
-While we don't support many Stage 1 proposals, there have been recent updates to the pipeline operator and to do expressions. Since we already support those proposals and the community is quite excited about them, we will update our implementations.
+尽管我们不支持许多第 1 阶段提案，但最近对管道运算符和 to do 表达式进行了更新。因为我们已经支持这些提案，并且社区对它们非常期待，所以我们将更新我们的实现。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
尽管我们不支持许多第 1 阶段提案，但最近对管道运算符和 do 表达式进行了更新。因为我们已经支持这些提案，并且社区也对它们非常期待，所以我们将继续更新我们的实现。
```
这里指的应该是下面这个提案
https://github.com/tc39/proposal-do-expressions

File: docs/roadmap.md [hunk 7]
```diff
@@ -1,150 +1,150 @@
...
 
-There are also other proposals (such as pattern matching) that we have not yet implemented because their champions expect to do significant changes to the syntax and semantics. However, we are closely following their development, and will implement them in Babel as soon as they stabilize a bit.
+还有其它提案（比如模式匹配）我们至今没有实现，因为它们的拥护者希望对语法和语义进行重大更改。然而，我们正在密切关注它们的发展，一旦它们稳定下来，我们就会在 Babel 中实现它们。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
还有其它提案（比如模式匹配）我们至今没有实现，因为它们的拥护者希望对语法和语义进行重大更改。不过，我们正在密切关注它们的发展，一旦它们稳定下来，我们就会在 Babel 中实现它们。
```

File: docs/roadmap.md [hunk 8]
```diff
@@ -1,150 +1,150 @@
...
-- a Babel "runner" (`@babel/cli`, `babel-loader`, `@rollup/plugin-babel`, etc)
+- 一个 Babel "运行器"（`@babel/cli`、`babel-loader`、`@rollup/plugin-babel` 等）
 
-Moving `@babel/preset-env` directly into `@babel/core` has two big advantages:
+将 `@babel/preset-env` 直接移入 `@babel/core` 有两大好处：
 
-- It will be easier to configure Babel in simple projects, you would only need to enable a `compileJS: true` option in `babel.config.json` (or it could even be the default in the future -- it can't be default as `@babel/eslint-parser` does not compile the source)
-- It will make sure that the plugin versions are in sync with the `@babel/core` version, avoiding most of the bugs caused by mismatched/incompatible packages versions
-- When we move to ESM, it will be hard to resolve and load plugin synchronously in `transformSync`. This prevents it from being a problem.
+- 在简单的项目中配置 Babel 将会更容易，你只需要在 `babel.config.json` 中启用 `compileJS: true` 选项（或者在将来它甚至可能是默认值 —— 它不能作为默认值是因为 `@babel/eslint-parser` 不编译源码）
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
- 在简单的项目中配置 Babel 将会更容易，你只需要在 `babel.config.json` 中启用 `compileJS: true` 选项（或者在将来它甚至可能是默认值 —— 目前它不能作为默认值是因为 `@babel/eslint-parser` 不编译源码）
```

File: docs/roadmap.md [hunk 9]
```diff
@@ -1,150 +1,150 @@
...
+- 它将确保插件版本与 `@babel/core` 版本保持同步，从而避免因为 不匹配/不兼容 的包版本而导致的大多数问题
+- 当我们迁移至 ES 模块时，将很难在 `transformSync` 中同步解析和加载插件。这样可以防止它成为一个问题。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
- 当我们迁移至 ES 模块时，将很难在 `transformSync` 中同步解析和加载插件。而这就可以解决前面的问题。
```

File: docs/roadmap.md [hunk 10]
```diff
@@ -1,150 +1,150 @@
...
 
-We have already decided to remove the older `core-js@2` support from `@babel/preset-env` in Babel 8. We also want to stop promoting a specific third-party polyfill, which might give our users the impression that it's part of Babel itself.
+我们已经决定从 Babel 8 的 `@babel/preset-env` 中移除较旧的 `core-js@2` 的支持。我们还希望停止推广特定的第三方 polyfill，因为这可能会给我们的用户留下这样的印象：它是 Babel 本身的一部分。
 
-This might happen in two different ways:
-- We just remove `core-js@3` from `@babel/preset-env` in Babel 8, encouraging users to migrate to `babel-plugin-polyfill-corejs3` (which is what `@babel/preset-env` internally uses starting from version 7.10.0)
-- We can keep `core-js@3` support in `@babel/preset-env`, but not migrate it to `@babel/core` when we'll move the transform plugins.
+这可能以两种不同的方式进行：
+- 我们只需从 Babel 8 的 `@babel/preset-env` 中移除 `core-js@3`，鼓励用户迁移到 `babel-plugin-polyfill-corejs3`（从 v7.10.0 开始，在 `@babel/preset-env` 内部使用）
+- 我们可以在 `@babel/preset-env` 中保留对 `core-js@3` 的支持，但在移动转译插件时，不能将其迁移到 `@babel/core`。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
- 我们可以在 `@babel/preset-env` 中保留对 `core-js@3` 的支持，但在移动转译插件时，不将其迁移到 `@babel/core`。
```

File: docs/roadmap.md [hunk 11]
```diff
@@ -1,150 +1,150 @@
...
 
-**Action Points**
+**行动要点**
 
-This goal can be split into two big tasks that can be done in parallel:
-- We need to identify *where* these optimizations can be useful by collecting real-world browserslist queries and by simulating how popular queries (for example, `defaults` or `>2%, not dead`) will evolve in the future.
-- We need to actually implement the necessary optimizations, making sure that they still work well with the other plugins (since they would highly increase the number of possible transform combinations).
+这个目标可以分为两个可以并行完成的大任务：
+- 我们需要通过收集真实的 browserslist 查询并模拟流行的查询（例如：`defaults` 或 `>2%, not dead`）在未来将如何演变，来确定这些优化在 *哪些方面* 有用。
+- 我们需要实际实现必要的优化，确保它们仍然可以与其他插件很好地协同工作（因为它们会极大地增加可能的转译组合的数量）。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
- 我们需要实际实现必要的优化，确保它们仍然可以与其他插件很好地协同工作（因为它们会极大地增加可能的转译组合数量）。
```

File: docs/roadmap.md [hunk 12]
```diff
@@ -1,150 +1,150 @@
...
 
-### Investigate new compiler `assumptions`
+### 调查新的编译器 `assumptions`
 
-In Babel 7.13.0 we introduced a new top-level [`assumptions`](https://babeljs.io/docs/en/options#assumptions) option, to formalize what the `loose` mode option does and offer more granular control to our users (since often they can only enable _some_ assumptions and not all of them).
+在 Babel 7.13.0 中，我们引入了一个新的顶层 [`assumptions`](https://babeljs.io/docs/en/options#assumptions) 选项来正式化 `loose` 模式选项的功能，并为我们的用户提供更精细的控制（因为他们通常只能启用 _某些_ 假设，而不是所有）。
 
-However, we only included options for assumptions we _already_ made when compiling in `loose` mode. We can now investigate what new assumptions our users might need.
+但是，我们只包含了在 `loose` 模式下编译时 _已经_ 做出的假设选项。我们现在可以调查我们的用户可能需要哪些新的假设。
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
但是，我们只包含了在 `loose` 模式下编译时 _已经_ 做出的假设选项。我们现在可以审查我们的用户可能需要哪些新的假设。
```

File: docs/roadmap.md [hunk 13]
```diff
@@ -1,150 +1,150 @@
...
 
-We can find which new assumptions we should implement, by:
-- Manually checking which features we compile to "non-obvious" output, which is usually caused by edge cases that many developers don't care about.
-- Ask for feedback from the community, since developers can test which assumptions work and which don't on their applications.
+我们可以通过以下方式找到我们应该实现的新假设：
+- 手动检查我们编译成“不明显”输出的特性，这通常是由许多开发人员不关心的边缘情况引起的。
+- 从社区寻求反馈，因为开发人员可以测试哪些假设对他们的应用有效，哪些假设无效。
 
-### Overhaul the Babel REPL
+### 大修 Babel REPL
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
### 检修 Babel REPL
```

File: docs/roadmap.md [hunk 14]
```diff
@@ -1,150 +1,150 @@
...
 
-The Babel REPL is a convenient playground to learn how Babel transpiles source code.
+Babel REPL 是一个方便学习 Babel 如何转译源码的 playground。
 
-Current limits:
+目前的限制：
 
-- The REPL does not support `assumptions` config. Although we have a dedicated per-assumption basis mini REPL on https://babel.dev/assumptions, currently we can not show how these `assumptions` might work together
-- The REPL does not support plugin options. Some plugins have required options, such as `@babel/plugin-proposal-record-and-tuple` and `@babel/plugin-proposal-decorators`
+- REPL 不支持 `assumptions` 配置。尽管我们目前在 https://babel.dev/assumptions 上有一个专门基于每个假设的迷你 REPL，但我们不能说明这些 `assumptions` 是如何协同工作的
```
Comment (KimYangOfCat) [inline docs/roadmap.md]:
```suggestion
- REPL 不支持 `assumptions` 配置。尽管我们目前在 https://babel.dev/assumptions 上有一个专门基于每个假设的迷你 REPL，但我们不能展示这些 `assumptions` 是如何协同工作的
```

**Review by KimYangOfCat — CHANGES_REQUESTED** [review]
感谢你的翻译，非常棒！麻烦再对其中一些校对意见进行修改吧，若有不同意见，欢迎讨论~

File: docs/roadmap.md (hunk 1 above)
Comment (htmlin) [reply docs/roadmap.md:9]:
```suggestion
这远不是我们将为 Babel 带来的所有新功能或重要更改的完整列表，但是，如果你对项目发展的大方向感兴趣，那么这是一个很好的总结。实际上，我们可能不能完成所有列出的要点，或者可能会将其中一些推迟到明年。其中一些有明确的起点和终点，而另一些则需要更多的研究或 [RFCs](https://github.com/babel/rfcs)。
```

**Comment by htmlin** [comment]
> 感谢你的翻译，非常棒！麻烦再对其中一些校对意见进行修改吧，若有不同意见，欢迎讨论~

done

**Review by KimYangOfCat — APPROVED** [review]
